utils/oomparser/oomparser.go

```diff
@@ -38,6 +38,9 @@ var firstLineRegexp *regexp.Regexp = regexp.MustCompile(
 // struct to hold file from which we obtain OomInstances
 type OomParser struct {
 	systemFile string
+
+	// Inotify event watcher.
+	watcher *inotify.Watcher
```

Comment:
Do we know if watching is better? For many systems, /var/log/messages gets updated more than once per second.